pkg/plugin/packetparser/packetparser_bpfel_arm64.go

```diff
@@ -49,12 +55,18 @@ type packetparserPacket struct {
 		Tsval  uint32
 		Tsecr  uint32
 	}
-	ObservationPoint uint8
-	TrafficDirection uint8
-	Proto            uint8
-	Flags            uint8
-	IsReply          bool
-	_                [3]byte
+	ObservationPoint  uint8
+	TrafficDirection  uint8
+	Proto             uint8
+	Flags             uint8
+	IsReply           bool
+	_                 [3]byte
```

Comment:
nit: I wonder if there is a way to arrange the struct so that we can minimize the paddings even more 🤔

Reply:
not unless we shrink/remove struct fields. @SRodi could any of the fields be combined? Eg. I magine `TrafficDirection` only has 2/3 values, and so does `Proto` and `IsReply`. Could at least one be merged into the `Flags` field, or could you create a ~Flags2 field for them?

Reply:
I agree but this is probably outside the scope of this PR. I captured the suggestions in https://github.com/microsoft/retina/issues/1121 and will implement in a separate PR.